### DISCUSSION TITLE
Installazione vlc_permanents_bookmarks.lua

### DISCUSSION THREAD

**Discussion by satsat47** [community/community#40620] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

premetto che sono al digiuno di programmazione e un pò duro di comprendonio. Sono un vecchietto di 75 anni e vorrei dividere i libri audio scaricati in parti tali da poterli inviare con whatsapp. Come, certamente, sapete whatsapp non accetta files superiori 16Mb.
Ho trovato Segnalibri permanenti VLC ed ho eseguito il programmino vlc_permanent_bookmakers.lua ed ho provato ad inserirlo nel programma Vlc, ma non funziona in quanto pur mettendo il segnalibro e chiudendo l'app, quando la riapro e richiamo il file in cui ho posto il segnalibro, inizia daccapo. anche se vado nei segnalibri salvati, lo trovo vuoto.
Per cortesia, c'è qualcuno di buona volontà, che mi spiega passo per passo come fare?
Grazie e saluti

[Installazione vlc_permanentsbookmarks.lua 2.pdf](https://github.com/community/community/files/10141474/Installazione.vlc_permanentsbookmarks.lua.2.pdf)
[Installazione vlc_permanentsbookmarks.lua 3.pdf](https://github.com/community/community/files/10141475/Installazione.vlc_permanentsbookmarks.lua.3.pdf)
[Installazione vlc_permanentsbookmarks.lua 4.pdf](https://github.com/community/community/files/10141476/Installazione.vlc_permanentsbookmarks.lua.4.pdf)
[Installazione vlc_permanentsbookmarks.lua 1.pdf](https://github.com/community/community/files/10141477/Installazione.vlc_permanentsbookmarks.lua.1.pdf)

**Comment by ossanna16** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
